### PR TITLE
[stable/kube-bench] Helm chart for kube-bench

### PR DESCRIPTION
**Pull request by goelankitt**
## Description
This helm chart creates a cronjob for running kube-bench. At this point, it only supports eks and gke.

## Checklist

- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [x] Github actions are passing

### CODE REVIEW

File: stable/kube-bench/templates/cron.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          volumes:
+            - name: var-lib-kubelet
+              hostPath:
+                path: "/var/lib/kubelet"
+            - name: etc-systemd
+              hostPath:
+                path: "/etc/systemd"
+            - name: etc-kubernetes
+              hostPath:
+                path: "/etc/kubernetes"
```
Comment (max-rocket-internet) [inline stable/kube-bench/templates/cron.yaml:39]: It's probably nicer to include the standard options that allow the user to customise the scheduling for their env. e.g. https://github.com/deliveryhero/helm-charts/blob/master/stable/k8s-event-logger/templates/deployment.yaml#L41-L52

File: stable/kube-bench/values.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+provider: eks
+
+cronjob:
+  schedule: "*/1 * * * *"
```
Comment (max-rocket-internet) [inline stable/kube-bench/values.yaml:4]: I think every minute is too frequent?? 😅

**Review by max-rocket-internet — CHANGES_REQUESTED** [review]
CI steps need to pass also

File: stable/kube-bench/values.yaml (hunk 2 above)
Comment (goelankitt) [reply stable/kube-bench/values.yaml]: yes it is. Making it monthly.

**Review by max-rocket-internet — APPROVED** [review]

**Pull request merged**